docs/pages/versions/unversioned/sdk/device.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Device
+---
+
+Provide information of devices on the application.
```

Comment:
Be more detailed: what kind of information? When would you want to use this module and continue reading the rest of this document?